malaria-app-android/src/main/java/com/peacecorps/malaria/services/RingService.java

```diff
@@ -0,0 +1,40 @@
+package com.peacecorps.malaria.services;
+
+import android.app.Service;
+import android.content.Intent;
+import android.media.Ringtone;
+import android.media.RingtoneManager;
+import android.net.Uri;
+import android.os.IBinder;
+import android.util.Log;
+
+public class RingService extends Service {
+    private Ringtone ringtone;
+
+    public RingService() {
+    }
+
+    @Override
+    public IBinder onBind(Intent intent)
+    {
+        return null;
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId)
+    {
```

Comment:
Same

Reply:
@anubhakushwaha Please check the commit 19b22b9e14c041060f1f17ad512a209ac324edd6 (Start and stop ring service) as I have squashed all the commits with minor changes into that one( as you wanted commits according to functionality)